README.md

```diff
@@ -19,28 +24,55 @@ This is an **experimental** fork of [Sebastián Ramírez's](https://github.com/t
 - [Help needed](#help-needed)
 - [Release notes](#release-notes)
 - [License](#license)
-  
-## Screenshots
 
-### App landing page
 
-![Landing page](img/landing.png)
+## Requirements
+
+Please make sure you have these installed before proceeding!
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+* [Python 3.11+](https://www.python.org/downloads/)
+* [Cookiecutter](https://cookiecutter.readthedocs.io/en/stable/) -- The Python Cookiecutter library.
+* [Hatch](https://hatch.pypa.io/latest/)
+* [Node.js](https://nodejs.org/en)
+* [MongoDB Compass](https://www.mongodb.com/products/tools/compass) -- To help visualize local/remote database easily.
+
+## QuickStart
+
+**NOTE: this will generate a local application that is not yet production-ready. Please go through all of the README information linked before creating a version to deploy to a production host**
```

Comment:
Maybe a more direct link to the relevant sections?

Reply:
I'd say here the `index` section has all those relevant pieces